arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -502,15 +586,33 @@ void StatisticsFeature::appendHistogram(
       std::to_string(v) + '\n';
   }
   result += name + "_count " + std::to_string(sum) + '\n';
+  if (v2) {
+    double v = slc.get("sum").getNumber<double>();
+    result += name + "_sum " + std::to_string(v) + '\n';
+  }
 }
 
-void StatisticsFeature::appendMetric(std::string& result, std::string const& val, std::string const& label) {
+std::string metricType(std::string const& type, bool v2) {
+  if (type.compare("counter") == 0 || type.compare("gauge") == 0 ||
+      type.compare("histogram") == 0) {
+    return type;
+  }
+  auto pos = type.find('/');
+  TRI_ASSERT(pos != std::string::npos);
+  return v2 ? type.substr(pos+1) : type.substr(0, pos);
+}
```

Comment:
Good idea, done. I did not do `string_view` as parameter, since this is more hazzle than it helps. `std::string const&` should be equally efficient (if not more efficient, since it only needs a single reference). For the return value, I used `string_view`, although it is more hazzle because there is no `operator+` for `std::string` with `std::string_view` and we are not in a performance critical path.